src/core/EdFi.DataManagementService.Core/Validation/DocumentValidator.cs

```diff
@@ -51,25 +52,38 @@ RequestMethod method
         EvaluationOptions validatorEvaluationOptions =
             new() { OutputFormat = OutputFormat.List, RequireFormatValidation = true };
 
+        // Parse the body just one time
+        JsonNode? parsedBody = JsonNode.Parse(frontendRequest.Body);
```

Comment:
Shouldn't you be using `PipelineContext.ParsedBody` and modifying it instead? `frontendRequest.Body` is no longer relevant once `ParseBodyMiddleware` runs. Make sure the tests are updated look at `context.ParsedBody` as well.